project/Build.scala

```diff
@@ -122,28 +122,45 @@ object Build {
   lazy val junitPlugin = MultiScalaProject("junitPlugin", file("junit-plugin"))
     .settings(compilerPluginSettings)
 
-  private val withSharedSources = Def.settings(
-    // baseDirectory = project/jvm/.<scala-version>
-    Compile / unmanagedSourceDirectories += baseDirectory.value.getParentFile.getParentFile / "src"
+  private val withSharedCrossPlatformSources = Def.settings(
```

Comment:
Was looking at this area but didn't know exactly how to solve it. The build is quite tricky.